satellite-slave/config/sample-settings.json

```diff
@@ -0,0 +1,39 @@
+{
+  "satellites": [
+    {"host": "satellite.master.1.example.com"},
+    {"host": "satellite.master.2.example.com"}
+  ],
+  "service": "mesos/slave/",
+  "comets": [
+    {"name": "satellite-slave.recipes/free-memory",
+     "params": [50],
+     "every": 60},
+    {"name": "satellite-slave.recipes/free-swap",
+     "params": [50],
+     "every": 60},
+    {"name": "satellite-slave.recipes/percentage-used",
+     "params": [90, "/tmp"],
+     "every": 60},
+    {"name": "satellite-slave.recipes/percentage-used",
+     "params": [90, "/var"],
+     "every": 60},
+    {"name": "satellite-slave.recipes/percentage-used",
+     "params": [90, "/tmp/mesos"],
+     "every": 60},
+    {"name": "satellite-slave.recipes/num-uninterruptable-processes",
+     "params": [10],
+     "every": 60},
+    {"name": "satellite-slave.recipes/load-average",
+     "params": [30],
+     "every": 60},
+    {"name": "satellite-slave.mesos.recipes/total-tasks-failed",
+     "params": [],
+     "every": 60},
+    {"name": "satellite-slave.mesos.recipes/total-tasks-finished",
+     "params": [],
+     "every": 60},
+    {"name": "satellite-slave.mesos.recipes/total-tasks-started",
+     "params": [],
+     "every": 60}
```

Comment:
Consider adding support for different every units besides seconds, perhaps something along the lines of:
```clojure
"every" : {"unit"  : "minutes" , "value" : 60}
```